README.rst

```diff
@@ -100,6 +100,43 @@ Install matchms from Anaconda Cloud with
 
   conda install --channel nlesc --channel bioconda --channel conda-forge matchms
 
+Example
+=======
+
+Below is a small example of using matchms to calculate the Cosine score between mass Spectrums.
+
+.. code-block:: python
+
+    from matchms.importing import load_from_mgf
+    from matchms.filtering import default_filters
+    from matchms.filtering import normalize_intensities
+    from matchms import calculate_scores
+    from matchms.similarity import CosineGreedy
+
+    file = load_from_mgf("tests/pesticides.mgf")
+
+    def my_filter(spectrum):
+        '''Clean and enhance the spectrums with a filter'''
+        spectrum = default_filters(spectrum)
+        spectrum = normalize_intensities(spectrum)
```

Comment:
```suggestion
        spectrum = normalize_intensities(spectrum) # a filter to normalize intensities
```

Reply:
Comment does not tell me anything more then the code, not adding

Reply:
Comment `Scale peak intensities to maximum of 1` added